common/src/main/scala/io/phdata/services/EmailService.scala

```diff
@@ -62,7 +65,13 @@ class EmailServiceImpl[F[_]: Effect](context: AppContext[F], workspaceService: W
       fromAddress = context.appConfig.smtp.fromEmail
     } yield {
       addressList.map(
-        recipient => context.emailClient.send("A New Workspace Is Waiting", email, fromAddress, recipient)
+        recipient =>
+          context.emailClient.send(
+            "A New Workspace Is Waiting",
+            EmbeddedImageEmail.create(email, "public/images/logo_big.png", "logo"),
```

Comment:
This might not work on the cluster. We need to add the image to the parcel so they show up at this path.